utils/get-changes.js

```diff
@@ -0,0 +1,46 @@
+import { Octokit } from "@octokit/rest";
+import { tokensStudioRepoParams } from "./github/api-client.js";
+
+const { GITHUB_TOKEN, HEAD_COMMIT } = process.env;
+
+export const ghClient = new Octokit({
+  auth: GITHUB_TOKEN,
+  baseUrl: "https://api.github.com",
+});
+
+export async function getChanges() {
+  try {
+    let levelsChanged = [];
+    const prNumber = HEAD_COMMIT.match(/#(\d+)/)?.[1];
+
+    if (!prNumber) {
+      return "";
+    }
+
+    const { data: files } = await ghClient.pulls.listFiles({
+      owner: tokensStudioRepoParams.owner,
+      repo: tokensStudioRepoParams.repo,
+      pull_number: prNumber,
+    });
+
+    files.forEach(({ filename }) => {
+      if (filename.includes("base.json")) {
+        levelsChanged.push("base");
+      }
+
+      if (filename.includes("brand/canvas.json")) {
+        levelsChanged.push("brand");
+      }
+
+      if (filename.includes("sys/")) {
+        levelsChanged.push("sys");
+      }
+    });
+
+    return levelsChanged.join(",");
+  } catch (error) {
+    throw new Error(`Error: Failed in get Changes. \n >>> ${error.message}`);
```

Comment:
Not blocking: idk if this would be helpful, but would it be helpful we logged out the levels changed in here. I'm just thinking if it does fail, maybe we can log out some additonal information that could be helpful.